build.gradle

```diff
@@ -1,14 +1,14 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
-    ext.kotlin_version = '1.3.61'
+    ext.kotlin_version = '1.6.21'
```

Comment:
Maybe we can bump it higher?

Reply:
Updated to 1.9.24, which forced `kotlin-android-extension` removal

Reply:
Ok thx for doing the change. Yeah `kotlin-android-extension` was deprecated a long time ago :D